### PR TITLE
feature/SKED-6421: Add information about security to Pull Request Template

### PR DESCRIPTION
**Pull request by gevalo1**
See JIRA: https://skedify.atlassian.net/browse/SKED-6421

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1 +1,7 @@
-> See JIRA https://skedify.atlassian.net/browse/SKED-
\ No newline at end of file
+> See JIRA https://skedify.atlassian.net/browse/SKED-
+
+<!--  
+In an effort to increase security awareness we want to remind ourselves to pay attention to security when writing code.
+👉 Mark this checkbox if you payed attention to security.
+-->
+- [ ] Security Awareness Checkbox 🛡️
```
Comment (kendepelchin) [inline .github/pull_request_template.md:7]: This will become "how to test" scenario very fast 😂

Comment (RobinMalfait) [reply .github/pull_request_template.md:7]:
```suggestion
- [x] Security Awareness Checkbox 🛡️
```

Saved you a click 😘

**Review by timothy-suy — CHANGES_REQUESTED** [review]
Q: do we have a written down version of the security patterns somewhere (confluence, ...)?
If so, I'd add the link here

**Comment by christophethelen** [comment]
@timothy-suy  we don't have a security patterns doc.
The goal for now is to ensure we indicate that we have at least spent some thoughts on security

**Comment by kendepelchin** [comment]
I don't see the benefit of this; I think this will quickly become "tick the box" instead of actually thinking about this.

**Review by timothy-suy — APPROVED** [review]

**Pull request merged**